app/routes.py

```diff
@@ -1,2 +1,51 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response
 
+class Planet:
+    def __init__(self, id, name, description, diameter):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.diameter = diameter
+ 
+planets = [Planet(1, 'Venus', 'The hottest planet',12104 ),
+           Planet(2,'Mercury', 'Has no natural satellite', 4879),
+           Planet(3,'Earth', 'The only planet where life is found', 12742 )
+     
+ ]
+
+planets_bp= Blueprint('planets', __name__,url_prefix='/planets')
+
+@planets_bp.route("", methods=['GET'])
+
+def handle_planets():
```

Comment:
Tthe decorator should come right before the function definition with no white space: https://www.flake8rules.com/rules/E304.html#:~:text=There%20should%20be%20no%20blank,the%20function%20it%20is%20decorating.